corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -4,6 +4,8 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
     var FormplayerFrontend = hqImport("cloudcare/js/formplayer/app"),
         Util = hqImport("cloudcare/js/formplayer/utils/util");
 
+    var selectedCaseIds = [];
```

Comment:
Similar to the comment below on `updateContinueButtonText`, this ought to be a property of the `CaseListView`, and if you pass that view as an option to the `CaseView`, you'll have access to update it from there.